docs/NooBaaNonContainerized/NooBaaCLI.md

```diff
@@ -458,8 +462,66 @@ noobaa-cli diagnose metrics
 The `gather-logs` command is used for extract NooBaa non containerized logs.
 Not implemented yet, running this command will fail with not implemented error.
 
+## Upgrade
 
-## Global Flags
+The `upgrade` command is being used for running config directory upgrade operations.
+- **[Start](#upgrade-start)**: Initiate config directory upgrade.
+- **[Status](#upgrade-status)**: Retrieve the in progress config directory upgrade status.
+- **[History](#upgrade-history)**: Retrieve the history information of past config directory upgrades.
+
+For more information about the config directory upgrade, See - [Upgrade](./Upgrade.md#online-upgrade-version--5180)
+
+### Upgrade Start
+
+The `upgrade start` command is used to start a config directory upgrade run.
+
+#### Usage
+```sh
+noobaa-cli upgrade start --expected_version --expected_hosts [--skip-verification] [--custom_upgrade_scripts_dir]
+```
+
+#### Flags -
+- `expected_version` (Required)
+    - Type: String
+    - Description: Specifies the upgrade's expected target version.
+    - Example - `--expected_version 5.18.0`
+
+- `expected_hosts` (Required)
+    - Type: String
+    - Description: Specifies the upgrade's expected hosts. String of hostnames separated by comma (,). 
+    - Example - `--expected_hosts hostname1,hostname2,hostname3`
+
+- `skip_verification`
+    - Type: Boolean
+    - Description: Specifies if NooBaa should skip verification of the expected hosts package version.  </br>
+    `WARNING:` can cause corrupted config directory files created by hosts running old code 
+
+- `custom_upgrade_scripts_dir`
+    - Type: String
+    - Description: Specifies a custom upgrade scripts directory. Used for running custom config directory upgrade scripts. </br>
+    `WARNING:` Specifying a custom upgrade scripts directory will initiate a non NooBaa official config directory upgrade and should not be used.
```

Comment:
@romayalon This statement is kind of contradictory, We have an option for customizing the upgrade script but here mentioned do not use it.